src/store/initialState.js

```diff
@@ -1,7 +1,14 @@
-import { products } from 'products';
```

Comment:
No reason to have this in another file, it's not that big. If your initial state is huge, or is dynamic (built via ssr or something) then it's ok to make its folder structure just as necessarily complex